airbyte-cdk/python/unit_tests/destinations/vector_db_based/document_processor_test.py

```diff
@@ -212,6 +216,200 @@ def test_process_multiple_chunks_with_relevant_fields():
     assert id_to_delete is None
 
 
+@pytest.mark.parametrize(
+    "label, text, chunk_size, chunk_overlap, splitter_config, expected_chunks",
+    [
+        (
+            "Default splitting",
+            "By default, splits are done \non multi newlines,\n\n then single newlines, then spaces",
+            10,
+            0,
```

Comment:
These tests were very helpful to readthrough as a demonstration of the various split algorithms. Nice work 👍